library/src/main/java/com/getstream/sdk/chat/utils/MessageListItemDiffCallback.java

```diff
@@ -29,11 +34,44 @@ public int getNewListSize() {
 
     @Override
     public boolean areItemsTheSame(int oldItemPosition, int newItemPosition) {
+
```

Comment:
this code is not needed (check on master)

Reply:
It's same on master. I think I didn't do anything in here.
https://github.com/GetStream/stream-chat-android/blob/master/library/src/main/java/com/getstream/sdk/chat/utils/MessageListItemDiffCallback.java#L31

Reply:
Oops, Sorry I see. I will remove it.